api/src/endpoints/index.js

```diff
@@ -3,6 +3,7 @@ import neo4jRoutes from 'endpoints/neo4j';
 import repoRoutes from 'endpoints/repository';
 import hpaRoutes from 'endpoints/hpaRna';
 import dataOverlayRoutes from 'endpoints/dataOverlay';
+import svgRoutes from 'endpoints/svgThumbnail';
```

Comment:
minor thing, but I would prob name this svgThumbnail instead of svgRoutes as the filename is svgThumbnail and it only relates to that

Reply:
I just don't like very long variable name unless necessary. Maybe I can change it to `svgThumbRoutes`.